package.json

```diff
@@ -3,7 +3,7 @@
   "name": "min-vid",
   "version": "0.1.9",
   "private": true,
-  "icon": "resource://min-vid/docs/images/gradient-logo.png",
+  "icon": "resource://min-vid/docs/images/dark-logo.png",
```

Comment:
hey @meandavejustice for some reason this icon doesn't show up in about:debugging or about:addons